BGAnimations/ScreenSelectProfile underlay/Input.lua

```diff
@@ -105,25 +104,16 @@ Handle.MenuDown = Handle.MenuRight
 Handle.DownRight = Handle.MenuRight
 
 Handle.Back = function(event)
-	if GAMESTATE:GetNumPlayersEnabled()==0 then
 		SCREENMAN:GetTopScreen():Cancel()
-	else
```

Comment:
This branch allowed to unjoin a player; since we can only have one, simply cancel into the previous screen.